src/Attorney.php

```diff
@@ -923,20 +1125,6 @@ public function numberOfEmployees($numberOfEmployees)
         return $this->setProperty('numberOfEmployees', $numberOfEmployees);
     }
 
-    /**
```

Comment:
This type is an organization only making offers but not getting offered.
https://schema.org/Attorney